apps/librelingo_json_export/tests/test_export.py

```diff
@@ -491,6 +512,22 @@ def test_matches_definitions_in_a_case_insensitive_way(self):
             "definition": "by a lot",
         }
 
+    def test_matches_definitions_with_spaces(self):
+        my_course = Course(
+            **{
+                **(fakes.course1._asdict()),
```

Comment:
maybe `fakes.customize` would make this easier/simpler? What do you think about it?

Reply:
I copied this from [the code directly above](https://github.com/kantord/LibreLingo/pull/1452/files#diff-ab9c50ea4369e0645fb5e151128d58dde9e38e4879661c5890f0be840cb5e650L481). If you'd like me to do it both places, I can switch to `fakes.customize`.

Reply:
> I copied this from the code directly above. If you'd like me to do it both places, I can switch to fakes.customize.

sounds good, but I'm also happy to hear your opinion about the usefulness of that function. Such opinions are very valuable for the project, as I'm not getting many code reviews myself 😸